src/main/java/com/google/sps/data/Instance.java

```diff
@@ -0,0 +1,88 @@
+package com.google.sps.data;
+
+import com.google.appengine.api.datastore.Entity;
+import java.util.Optional;
+import com.google.sps.data.EntityConstants;
+
+/** Class that represents an instance. */
+public class Instance implements InstanceInterface {
+  
+  // The name of this instance.
+  private final String instanceName;
+
+  // The session associated with this instance.
+  private Optional<String> sessionId;
+
```

Comment:
Dont we also want to track the the state of the VM? https://cloud.google.com/compute/docs/instances/instance-life-cycle
So we can tell if a VM is running or not or in any other state. We'll also want a background task taht periodically updates this.